README.md

```diff
@@ -4,6 +4,7 @@ Wikidata bietet sich als die idealle Datenbank für Informationen zu österreich
 
 * [Wikidata.org](http://wikidata.org/)
 * [SPARQL Query Service](https://query.wikidata.org/)
+* [Project Page on Wikidata for Austrian politics](https://www.wikidata.org/wiki/Wikidata:WikiProject_Politics_Austria/)
```

Comment:
lose the trailing slash, the link won't work with it.

Reply:
Thanks for the catch.  Fixed.